conf/.htaccess.uploads.fast404

```diff
@@ -2,8 +2,9 @@
 # WordPress "File not found" page rendering.
```

Comment:
Major: If this snippet is placed into the main .htaccess file now, then developers need to be informed that they must not touch the lines as they would be duplicated otherwise.

The common way to do this in WordPress is to start the leading comment with `BEGIN` and add a trailing comment after the snippet/block starting with `END`.